packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -235,8 +235,13 @@ const prompts = () => {
         },
         {
             name: 'siteId',
-            message: "What is your site's ID (examples: RefArch, SiteGenesis) in Business Manager?",
-            validate: validSiteId
+            message:
+                "What is your site's ID (examples: RefArchGlobal, SiteGenesis) in Business Manager?",
+            validate: validSiteId,
+            default: () => {
```

Comment:
i'm wondering what does the `default` mean for this question? Should it have a default at all? 🤔

Reply:
This is what I understand by "pre-set to be `RefArchGlobal`":

> Update Generator:
When we ask for the site ID the example and the pre-set to be “RefArchGlobal”.

https://www.figma.com/file/wzb7mqq8TOfEFDz5a4AbX9/Localization-%2F-Multi-site?node-id=122%3A97

Reply:
Talking with Sejal, we decided to leave the changes on the generator for a future PR.